pkg/frontend/clustermanager_get.go

```diff
@@ -34,12 +34,14 @@ func (f *frontend) _getClusterManagerConfiguration(ctx context.Context, log *log
 	vars := mux.Vars(r)
 
 	doc, err := f.dbClusterManagerConfiguration.Get(ctx, r.URL.Path)
-	switch {
-	case cosmosdb.IsErrorStatusCode(err, http.StatusNotFound):
-		return nil, api.NewCloudError(http.StatusNotFound, api.CloudErrorCodeResourceNotFound, "", "The Resource '%s/%s/%s/%s' under resource group '%s' was not found.",
-			vars["resourceType"], vars["resourceName"], vars["ocmResourceType"], vars["ocmResourceName"], vars["resourceGroupName"])
-	case err != nil:
-		return nil, err
+	if err != nil {
+		switch {
+		case cosmosdb.IsErrorStatusCode(err, http.StatusNotFound):
```

Comment:
Would an http 401 or 403 be possible here as well?

Reply:
In those cases the caller would get a generic error (500?) rather than a `CloudErrorCodeResourceNotFound`.

Reply:
our web server wouldn't return a 401/403 at this point in the code, ARM handles the authentication and authorization, and we have middleware that performs some authorization as well, see https://github.com/Azure/ARO-RP/blob/8fc2fddea4eee96c8ff05730368cd05292e128eb/pkg/frontend/middleware/authenticated.go#L30-L32